packages/ubus-lime-voucher/files/usr/libexec/rpcd/lime-voucher

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env lua
+--[[
+Copyright 2018 Marcos Gutierrez <gmarcos87@gmail.com>
```

Comment:
@luandro don't forget to add yourself to the list of developers.

Reply:
Okie :)